shared/experimental/qlpack.yml

```diff
@@ -0,0 +1,7 @@
+name: codeql/experimental
```

Comment:
If a new qlpack is added, I think it needs to be added to the list [here](https://github.com/github/semmle-code/blob/main/buildlib.py#L872). Do you have access to the internal repo?

Reply:
Opened a PR: https://github.com/github/semmle-code/pull/53077.

Reply:
As PR discussion on slack, we should consider renaming this to `codeql/quantum`.